Maskbook/Extension/UserDefaults+extension.swift

```diff
@@ -246,18 +246,19 @@ final class UserDefaultSettings {
         passwordExpiredDate = Date().addingTimeInterval(30 * 60)
         os_log(.debug, "[wallet-password] reset wallet password expired date %s", passwordExpiredDate?.description ?? "nil")
     }
-
-    func isPasswordExpried() -> Bool {
-        guard let expiredDate = passwordExpiredDate else {
+    
+    func isPasswordExpried(_ expiredDate: Date? = nil) -> Bool {
+        // fixed: Simultaneous accesses to 0x1068319f0, but modification requires exclusive access.
+        guard let expiredDate = expiredDate ?? passwordExpiredDate else {
             return true
         }
 
         return expiredDate < Date()
     }
 
     // Risk warning moved out from plugins view, temporarily set this default to true
-    @ReactiveUserDefault(key: .pluginRiskWarningAwared, defaultValue: true)
-    var pluginRiskWarningAwared: Bool
+    @ReactiveUserDefault(key: .pluginRiskWarningAwared, defaultValue: [])
+    var pluginRiskWarningAwared: [String]
 
```

Comment:
Did `pluginRiskWarningAwared` have value for online users?

Reply:
Each wallet address needs to confirm the warning. `pluginRiskWarningAwared` is an array contains addresses that confirmed warning.

Reply:
I mean `pluginRiskWarningAwared ` previous value is `bool`.
You change it's type to array.
If online user already have a bool value, It will crash when get `pluginRiskWarningAwared ` array value